coverage/lcov.info

```diff
@@ -1,174 +1,137 @@
 TN:
```

Comment:
Let's ignore the entire Coverage folder, we should rely on a CI to move that to Coveralls.

Reply:
got it!